src/shared/components/Table/TableHead/TableHead.types.ts

```diff
@@ -3,6 +3,7 @@ import { HeadCell, Order } from 'shared/types/table';
 import { UiType } from '../Table.types';
 
 export type TableHeadProps = {
+  className?: string;
```

Comment:
I'm not seeing where you've introduced code that uses this new `className` prop. Just curious of its purpose.

Reply:
It's used [here](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1657/commits/1baa9aab8849da8cb175ba9b57c86cd18bde5c13#diff-73665bee8296711d9fe42d0b7e90eb2d25901aa022259d7acd6b7a4465c14098R30-R33), to allow the classname that the StyledComponent creates to be passed to the StyledComponent that wraps the MUI TableHead.

Reply:
Oh interesting, didn't realize that would be needed. TIL, thx!